docs/faq.md

```diff
@@ -38,9 +38,9 @@ For example, let's say that a preset instead used the webpack object configurati
 config.plugins.push(new webpack.EnvironmentPlugin(['NODE_ENV']));
 ```
 
-If you wanted to extend this plugin in your own project to add more environment variables, you would most likely
-resort to either adding a new instance of the `EnvironmentPlugin` by requiring webpack yourself, or looping through
-webpack's plugin array, removing the plugin, and re-instantiating it with your own arguments.
+If you wanted to extend this plugin in your project to add more environment variables, you would most likely
+resort to either adding a new instance of the EnvironmentPlugin by requiring webpack yourself, or looping through
+webpack's plugin array, removing the plugin, and re-instantiating it with your arguments.
```

Comment:
These two changes regress the readability IMO. Reverting back:
```suggestion
resort to either adding a new instance of the `EnvironmentPlugin` by requiring webpack yourself, or looping through
webpack's plugin array, removing the plugin, and re-instantiating it with your own arguments.
```

Reply:
In this, the use of 'own' isn't necessary as its been stated before.